### PR TITLE
Update data tracker location updater description

### PR DESCRIPTION
**Pull request by chiaberry**
The description on this dag was a duplicate of another dag's description. 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Add note to 1PW secrets moved to API vault and check for duplicates

### CODE REVIEW

**Review by frankhereford — APPROVED** [review]
This is super helpful, thank you Chia!

File: dags/atd_knack_data_tracker_location_updater.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 with DAG(
     dag_id=f"atd_knack_data_tracker_location_updater",
-    description="Assigns signal records to CSR issues in data tracker based on CSR location",
+    description="With data from AGOL, update the Signal Engineer Area field in Knack ",
```
Comment (johnclary) [inline dags/atd_knack_data_tracker_location_updater.py]: It is actually [a few more things](https://github.com/cityofaustin/atd-knack-services/blob/3e3a4817df90b637d99a59dfa94b965e29504323/services/config/locations.py#L18) besides engineer areas 📍

Comment (chiaberry) [reply dags/atd_knack_data_tracker_location_updater.py]: 😬 
how about "with data from AGOL, update signal location information in knack"?

Comment (johnclary) [reply dags/atd_knack_data_tracker_location_updater.py]: sounds great! 🙏

**Review by johnclary — APPROVED** [review]
🙏

**Review by mddilley — APPROVED** [review]
📚 🐳 🚢 🙏

**Pull request merged**